site/docs/src/json/jwt/vend-post-request.json

```diff
@@ -0,0 +1,8 @@
+{
+  "claims": {
+    "id":"cksknqcp90002au664j1j1cul",
```

Comment:
Does it make sense to have this be a FusionAuth compatible UUID? That seems to be a major use case (because then you can store things under the UUID name, and create a user with that UUID later).

Reply:
Yes, that is a good idea.  I will change to match.